libs/net/keycloak/Models/ResourceAccessModel.cs

```diff
@@ -17,13 +17,13 @@ public class ResourceAccessModel
     /// <summary>
     /// get/set - Realm management claims.
     /// </summary>
-    [JsonPropertyName("tno-app")]
+    [JsonPropertyName("mmi-app")]
```

Comment:
I'd love to come up with a way where these were not hardcoded.  It would have made this PR easier.